modules/contact/src/Plugin/Fz152/Contact.php

```diff
@@ -19,30 +72,22 @@ public function getSettingsPage() {
     return [
       'path' => 'contact',
       'title' => 'Contact',
-      'form' => '\Drupal\fz152_contact\Form\Fz152ContactSettings',
+      'form' => Fz152ContactSettings::class,
       'weight' => 5,
     ];
   }
 
-  /**
-   * {@inheritdoc}
-   */
-  protected function getContactBundles() {
-    $bundle_info = \Drupal::service('entity_type.bundle.info');
-    return $bundle_info->getBundleInfo('contact_message');
-  }
-
   /**
    * {@inheritdoc}
    */
   public function getForms() {
     $forms = [];
-    $contact_bundles = $this->getContactBundles();
-    foreach ($contact_bundles as $bundle_name => $bundle_info) {
-      $config = \Drupal::config("fz152_contact.settings.$bundle_name");
+    $contact_forms = $this->bundleInfo->getBundleInfo('contact_message');
+    foreach (array_keys($contact_forms) as $form) {
+      $config = $this->configFactory->get("fz152_contact.settings.$form");
```

Comment:
Better to move that settings into third party config inside contact form bundles like http://dgo.to/conact_storage module doing

Reply:
Filed https://www.drupal.org/project/fz152/issues/2927035